### DISCUSSION TITLE
Importance of hierarchical clustering

### DISCUSSION THREAD

**Discussion by j-jsmith** [MaartenGr/BERTopic#1203] [Q&A] [answered]
Hi Maarten, hi community!

I had a look at the hierarchical clustering of my tweets and it seems that topics have been clustered together that don't seem to really fit. 

For example, in the image attached, we can see that the topic "minimum wage" has been clustered with "climate crisis" (colour purple) or "voting" with "afghanistan" (colour green). 

I'm guessing that this is problematic, right? Would merging topics be the solution? Is there some way to isolate topics, like in the case of tweets mentioning "afghanistan", there doesn't seem to be any other topic that could be related to it, thus it seems to be a cluster of its own, how can I adapt this in the model? 

Thanks so much in advance! 

![Hierarchical clustering](https://user-images.githubusercontent.com/123753926/233665546-c0c488cf-b959-4658-9f09-7fb46db3781b.png)

**Comment by MaartenGr** [comment, marked as answer]
Actually, the underlying dataframe that you use is working correctly but there is a known issue with the visualization of that dataframe. It is fixed in the main branch (#1173), so if you install BERTopic from its latest commit, then it should work correctly.

**Reply by j-jsmith** [reply]
Super, thanks so much Maarten for fast and helpful responses!
